articles/aks/security-bulletins/overview.md

```diff
@@ -24,6 +24,47 @@ These updates cover security information related to the following AKS components
 
 ---
 
+## AKS-2025-008 CVE-2025-4563: Nodes can bypass dynamic resource allocation authorization checks
+
+**Published Date**: June 19, 2025
+
+### Description
+
+A vulnerability has been identified in the Kubernetes `NodeRestriction` admission controller where nodes can bypass dynamic resource allocation authorization checks. When the `DynamicResourceAllocation` feature gate is enabled, the controller validates resource claim statuses during pod status updates but fails to do so during pod creation. This allows a compromised node to create mirror pods that access unauthorized dynamic resources, potentially leading to privilege escalation.
+
+This issue has been assigned **CVE-2025-4563** and rated **Low severity (CVSS 3.1 score: 2.7)** due to the feature being disabled by default and the niche nature of the attack scenario.
+
+### References
+
+- [CVE-2025-4563](https://github.com/kubernetes/kubernetes/issues/132151)
+
+### Affected Components
+
+#### [**AKS Cluster**](#tab/aks-cluster)
+
+**Affected Versions**
+
+- kube-apiserver: v1.32.0 – v1.32.5
+- kube-apiserver: v1.33.0 – v1.33.1
+
+**Resolutions**
+
+- Fixed in kube-apiserver versions:
+  - v1.32.6 and later
+  - v1.33.2 and later
+
+- Upgrade to kube-apiserver v1.32.6 or v1.33.2 and later, where the issue is resolved.
```

Comment:
we probably need to rephrase this part as the in use 1.32.x 1.33.x are also not impacted. @riyac12